test/sql/order/test_limit.test

```diff
@@ -67,3 +63,74 @@ Hello World
 # can only limit by integers
 statement error
 select 1 limit date '1992-01-01';
+
+
+# Test limits with no constants
```

Comment:
Extra test cases: 
* Both LIMIT and OFFSET together with subqueries
* Subqueries that return zero rows
* Limit of NULL, offset of NULL
* Subqueries that return NULL
* Subqueries that return negative values
* Subqueries that return strings that can't be converted to integers